src/components/Badge/Badge.visual.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TPAComponentsProvider } from '../TPAComponentsConfig';
+import {Badge, BADGE_PRIORITY} from './';
+import {VisualContainerElement} from "../../../test/visual/VisualContainerElement";
+
+class BadgeVisual extends React.Component<any> {
+  static defaultProps = {
+    mobile: false,
+  };
+
+  render() {
+    const { mobile } = this.props;
+
+    return (
+      <TPAComponentsProvider value={{ mobile }}>
```

Comment:
We can also remove the `TPAComponentsProvider` here, since there's no `mobile` state.
I know it's a bit of a pain.
We're considering removing the `mobile` state from the generator, since it's causing so much refactoring of the code.